Modules/_struct.c

```diff
@@ -1832,9 +1832,6 @@ s_pack(PyObject *self, PyObject **args, Py_ssize_t nargs, PyObject *kwnames)
             "pack expected %zd items for packing (got %zd)", soself->s_len, nargs);
         return NULL;
     }
-    if (!_PyArg_NoStackKeywords("pack", kwnames)) {
```

Comment:
Indeed, I did not know this could be called via a different patch. Thanks for spotting this.